app/app.go

```diff
@@ -82,102 +89,155 @@ func (a *app) HealthCheck(ctx context.Context) error {
 	return a.store.Ping(ctx)
 }
 
-func (a *app) GetSettings(ctx context.Context) (model.Settings, error) {
-	return a.store.GetSettings(ctx)
+func (a *app) GetIntegrations(ctx context.Context) ([]model.Integration, error) {
+	return a.store.GetIntegrations(ctx)
+}
+
+func (a *app) GetIntegrationById(ctx context.Context, id uuid.UUID) (*model.Integration, error) {
+	integration, err := a.store.GetIntegrationById(ctx, id)
+	if err != nil {
+		switch cause := errors.Cause(err); cause {
+		case store.ErrObjectNotFound:
+			return nil, ErrIntegrationNotFound
+		default:
+			return nil, err
+		}
+	}
+	return integration, err
+}
+
+func (a *app) CreateIntegration(ctx context.Context, integration model.Integration) error {
+	err := a.store.CreateIntegration(ctx, integration)
+	if err == store.ErrObjectExists {
+		return ErrIntegrationExists
+	}
+	return err
 }
 
-func (a *app) SetSettings(ctx context.Context, settings model.Settings) error {
-	return a.store.SetSettings(ctx, settings)
+//nolint: lll
+func (a *app) GetDeviceIntegrations(ctx context.Context, deviceID string) ([]model.Integration, error) {
```

Comment:
Instead of using nolint: lll, can you please break it down into multiple lines?
That's the style that we used in other repositories.

```
func Name(
    param1,
    param2
) (return) {
...
```